src/main/java/com/spotify/docker/BuildMojo.java

```diff
@@ -601,6 +664,30 @@ private void createDockerFile(final String directory, final List<String> filesTo
     return allCopiedPaths;
   }
 
+  /**
+   * When running on windows, paths are backslash separated,
+   * we need to convert them to unix '/' separated paths.
+   *
+   * @param path
+   *            object (system specific)
+   * @return unix style path
+   */
+  private String unixPath(Path path) {
+    StringBuilder sb = new StringBuilder();
```

Comment:
TBH I simply lifted this function without looking at it too close. Now I'm wondering the same.